### ISSUE TITLE
XrdSysE2T commit breaks compilation

### ISSUE THREAD

**Issue by adriansev** [xrootd/xrootd#1078]
Building the python bindings i see this :   
```
In function ‘const char* XrdSysE2T(int)’:
cc1plus: error: function may return address of local variable [-Werror=return-local-addr]
/home/adrian/GIT-REPOS/xrootd/src/XrdSys/XrdSysE2T.cc:99:16: note: declared here
   99 |    std::string eTxt(e2sMap[errcode]);
      |                ^~~~
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/XrdUtils.dir/build.make:89: src/CMakeFiles/XrdUtils.dir/XrdSys/XrdSysE2T.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:468: src/CMakeFiles/XrdUtils.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
Traceback (most recent call last):
  File "./setup.py", line 136, in <module>
    'bdist_wheel': CustomWheelGen
  File "/home/adrian/.local/lib/python3.7/site-packages/setuptools/__init__.py", line 143, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib64/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib64/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib64/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "./setup.py", line 96, in run
    raise Exception( 'Install step failed!' )
Exception: Install step failed!
```

**Comment by simonmichal** [comment]
Fixed in b23edfb2a3e6ef803f89bafe0a1a5d5a8741bec7.

**Comment by abh3** [comment]
OK, thanks. I suppose I'll not use an anonymnous name space for that 
variable.

Andy
